react/features/base/toolbox/components/ToolboxItem.web.js

```diff
@@ -21,23 +21,40 @@ export default class ToolboxItem extends AbstractToolboxItem<Props> {
      */
     _renderItem() {
         const {
+            disabled,
             onClick,
-            showLabel
+            showLabel,
+            tooltipPosition
         } = this.props;
+        const className = showLabel ? 'overflow-menu-item' : 'toolbox-button';
         const props = {
             'aria-label': this.accessibilityLabel,
-            className: showLabel ? 'overflow-menu-item' : 'toolbox-button',
-            onClick
+            className: className + (disabled ? ' disabled' : ''),
+            onClick: disabled ? undefined : onClick
         };
         const elementType = showLabel ? 'li' : 'div';
+        const useTooltip = this.tooltip && this.tooltip.length > 0;
         // eslint-disable-next-line no-extra-parens
-        const children = (
+        let children = (
             <Fragment>
                 { this._renderIcon() }
-                { showLabel && this.label }
+                { showLabel && <span>
```

Comment:
why the span?

Reply:
Because as far as i saw, these 2 components get generated in a div and clear text inside a div is invalid html

Reply:
I still don't understand why you need to change the layout. The tooltip was shown when hovering the icon. Why did you change that?

Reply:
Because we need to show the tooltip when the mouse is hovering on the label too

Reply:
Ah, that's why! I wish you had made a separate commit for it, because it's not related to any of the rest, and it would have made me not scratch my head :-)

Reply:
Yeah, sorry, you may be right